### DISCUSSION TITLE
Too few arguments to function PHPStan\PhpDocParser\Parser\ConstExprParser::__construct(), 0 passed in

### DISCUSSION THREAD

**Discussion by ghost** [api-platform/core#6793] [General]
**Too few arguments to function PHPStan\PhpDocParser\Parser\ConstExprParser::__construct(), 0 passed in**  this issue occur while composer phar update now i downgrade the package to 3.0  can you suggest a better solution to resolve this problem in latest version itself

**Comment by soyuka** [comment]
#6789

**Comment by ghost** [comment]
Still, there are no changes.
